src/pages/validator/external-chains/frax.mdx

```diff
@@ -0,0 +1,342 @@
+        "number": 323353
+      },
+      "sequenceNumber": 2
+    },
+    "finalized_l2": {
+      "hash": "0x43e19e7bed3d205aec15228b3f689c9d5f40920b9ff8ea3bd325f1e3fe70b757",
+      "number": 30709,
+      "parentHash": "0x87d7b203a4d868ea4aff0fa8fc60b98977ad5cc1f6fc260b3d0f3f577a4526dd",
+      "timestamp": 1700058794,
+      "l1origin": {
+        "hash": "0xfb429c1a5797d239b19d3101cf72683a4366f0ad2f0364c29786c32b629a878c",
+        "number": 323313
+      },
+      "sequenceNumber": 3
+    },
+    "pending_safe_l2": {
+      "hash": "0x7b7ed381aac9cec2a43b37c30779832cb8a82e044e38b4ada0f1c90719bd3d50",
+      "number": 30955,
+      "parentHash": "0x882503037e902b2476e1d4c308a895cc3b936c5ee19b45361dc26c9631d29ae0",
+      "timestamp": 1700059286,
+      "l1origin": {
+        "hash": "0x49ee7a7398bc5010b48de111fc0ebc99d63d38ac8233202c913646d901e15213",
+        "number": 323353
+      },
+      "sequenceNumber": 2
+    },
+    "queued_unsafe_l2": {
+      "hash": "0xeeb475b9876628f879c71e47d22e361c468b751b196a374b7a7f1a8f823aa7bc",
+      "number": 3516054,
+      "parentHash": "0xfdabc208e76735b29a2730d61923990e88f1e7f2f0b42fda0e91b5b813e2113b",
+      "timestamp": 1707029484,
+      "l1origin": {
+        "hash": "0x25c332267c9b3d7d3df832e2d98e31fbcc6e113a0494ed55673cbe456df7c32e",
+        "number": 873680
+      },
+      "sequenceNumber": 5
+    },
+    "engine_sync_target": {
+      "hash": "0x7b7ed381aac9cec2a43b37c30779832cb8a82e044e38b4ada0f1c90719bd3d50",
+      "number": 30955,
+      "parentHash": "0x882503037e902b2476e1d4c308a895cc3b936c5ee19b45361dc26c9631d29ae0",
+      "timestamp": 1700059286,
+      "l1origin": {
+        "hash": "0x49ee7a7398bc5010b48de111fc0ebc99d63d38ac8233202c913646d901e15213",
+        "number": 323353
+      },
+      "sequenceNumber": 2
+    }
+  }
+}
+```
+
+You can compare the block height on your node with explorer ([mainnet](https://explorer.mainnet.frax.com) or [testnet](https://explorer.testnet.frax.com)),
+use your RPC node only when it has caught up with the latest block height.
+
+#### Sync Status of Frax optimism Execution Node
+
+You can check your status by connecting to geth console
+
+```bash
+docker exec -it fraxtal-node-$ENVIRONMENT-geth-1 geth attach http://localhost:8545
+> eth.blockNumber
+35065 # it will show you the latest block - number here is just an example
+
+> eth.syncing
+false
+```
+
+## Configure vald
+
+In order for `vald` to connect to your Fraxtal node, your `rpc_addr` should be exposed in
+vald's `config.toml`
+
+<tabs>
+    <tab-item title="Mainnet">
+        ```bash
+        [[axelar_bridge_evm]]
+        name = "frax"
+        rpc_addr = "http://IP:PORT"
+        start-with-bridge = true
+        ```
+    </tab-item>
+    <tab-item title="Testnet">
+        ```bash
+        [[axelar_bridge_evm]]
+        name = "frax"
+        rpc_addr = "http://IP:PORT"
+        start-with-bridge = true
+        ```
+    </tab-item>
+</tabs>
+
+### Common Node Operations
```

Comment:
these can be removed since we duplicate on all pages. we can link to docker compose example from the ethereum page next near the top, cc @ffe9f8

Reply:
Added a link